packages/cli/cloud/src/services/cli-api.ts

```diff
@@ -10,13 +10,15 @@ import packageJson from '../../package.json';
 export const VERSION = 'v1';
 
 export type ProjectInfos = {
+  id: string;
   name: string;
-  nodeVersion: string;
-  region: string;
+  displayName?: string;
+  nodeVersion?: string;
+  region?: string;
   plan?: string;
   url?: string;
 };
-export type ProjectInput = Omit<ProjectInfos, 'id'>;
+export type CreateProjectInput = Omit<ProjectInfos, 'id'>;
```

Comment:
renamed this because it's the type we use for the input of the project creation

Reply:
I believe the project would have the same input everywhere it is modified, not only in create? 🤔